build.fsx

```diff
@@ -69,6 +69,8 @@ Target "NuGet" (fun _ ->
         | "ApiCheck" ->
             CopyFile libDir (buildDir @@ "ApiCheck.dll")
             CopyFile libDir (buildDir @@ "ApiCheck.XML")
+            CopyFile libDir (buildDir @@ "YamlDotNet.dll")
+            CopyFile libDir (buildDir @@ "YamlDotNet.XML")
```

Comment:
Agreed. Shouldn't we do this for the CommandLineParser dependency on ApiCheck.Console as well then?